src/PIL/DdsImagePlugin.py

```diff
@@ -98,6 +98,8 @@
 DXGI_FORMAT_R8G8B8A8_TYPELESS = 27
 DXGI_FORMAT_R8G8B8A8_UNORM = 28
 DXGI_FORMAT_R8G8B8A8_UNORM_SRGB = 29
+DXGI_FORMAT_BC4_TYPELESS = 79
+DXGI_FORMAT_BC4_UNORM = 80
```

Comment:
A note that you may or may not care about - I'm hoping to get #6486 merged before the next release. If it is merged in time, then I would remove these new constants in favour of enums. You've absolutely done the right thing here, no need to change this PR, just letting you know that if you had plans to use these constants externally, things may not play out that way.

Reply:
#6486 has now been merged